src/hera/workflow.py

```diff
@@ -52,6 +53,8 @@ class Workflow:
         If you create a WorkflowTemplate resource either clusterWorkflowTemplate or not (clusterScope attribute bool)
         you can reference it again and again when you create a new Workflow without specifying the same tasks and
         dependencies. Official doc: https://argoproj.github.io/argo-workflows/fields/#workflowtemplateref
+    host_aliases: Optional[List[HostAlias]] = None
```

Comment:
Perhaps this can be added to the cron workflow as well? 🤔

Reply:
added to the cron workflow 👍 
https://github.com/argoproj-labs/hera-workflows/pull/149/commits/42c52b57491c3cf16b7f0f14e272fb4572e2085e